### ISSUE TITLE
Setting model data type

### ISSUE THREAD

**Issue by jakeryansmith** [vuejs/Discussion#322]
I have a radio button I want to use to hide or show another element.

``` <input type="radio" name="track_inventory" v-model="trackInventory" value="1" > Yes ```
but for some reason I can't use this code 
``` v-if="value" ``` instead I have to use ``` v-if="value === '1'  ```

Do I need to cast the value of 'trackInventory' to a boolean? If so what is the cleanest way to do that? 

Thanks!

**Comment by swift1** [comment]
http://jsfiddle.net/1onz2ho7/3/

**Comment by jakeryansmith** [comment]
Right, I figured I could use a computed property. I was wondering if there is a way to mutate model values themselves? Or is using compute properties the best way to achieve this?

**Comment by swift1** [comment]
You can also use a filter:
http://jsfiddle.net/1onz2ho7/4/

I don't know if a filter causes more or less computations than a computed though, but I guess a filter might be the "right" way to do it in this case.

**Comment by jakeryansmith** [comment]
I can always just do a string comparison I guess too. ``` v-if="trackInventory === 'true' ``` 
Thanks for your help!

**Comment by swift1** [comment]
A third solution is to use a parseInt:
http://jsfiddle.net/1onz2ho7/5/

Less code, and perhaps the best solution.

**Comment by jakeryansmith** [comment]
Ahh! That's what I'm looking for.  I was trying ```v-show="Boolean(trackInventory)"``` but it wasn't working. parseInt works great. Thanks!

**Comment by mage3k** [comment]
@swift1 :+1:
